scss/main/_item.scss

```diff
@@ -129,69 +110,56 @@
                             position: absolute;
                             pointer-events: none;
                         }
-
                         &:before {
                             border-bottom-color: #c4c4c4;
                             border-width: 9px;
-
                             .en & {
                                 left: 19px;
                             }
-
                             .he & {
                                 right: 19px;
                             }
                         }
-
                         &:after {
                             border-bottom-color: #f5f5f5;
                             border-width: 6px;
-
                             .en & {
                                 left: 22px;
                             }
-
                             .he & {
                                 right: 22px;
                             }
                         }
-
                         &__title {
                             font-size: 20px;
                             margin-bottom: 8px;
                         }
-
-                        &__view, &__remove {
+                        &__view,
+                        &__remove {
                             text-decoration: underline;
-                            pointer: cursor;
+                            cursor: pointer;
```

Comment:
this looks like a different change from the resize font - if it was intended, please document it